README.md

```diff
@@ -23,68 +23,77 @@ Never submit security-related bugs through a Github Issue or by email.
 
 ## Requirements
 
-* Node 6.x LTS
+* You need [Node](https://nodejs.org/) 6.x which is the current
+  [LTS](https://github.com/nodejs/LTS) (long term support) release.
+* To make life easier, also install [yarn](https://yarnpkg.com/en/).
```

Comment:
Since we're pushing yarn as the command runner and dep installer it would make sense to make it sound a bit less optional (or alternatively clarify how to run commands with `npm run` in its place)